saved-sites/saved-sites-store/src/main/java/com/duckduckgo/savedsites/store/SavedSitesEntitiesDao.kt

```diff
@@ -97,7 +97,7 @@ interface SavedSitesEntitiesDao {
         lastModified: String = DatabaseDateFormatter.iso8601(),
     )
 
-    @Query("select * from entities order by rowid desc limit 1")
+    @Query("select * from entities limit 1")
```

Comment:
unsure, so just highlighting this, if we expect "lastmodified" not sure why we remove `order by`